package.json

```diff
@@ -22,9 +22,11 @@
   },
   "main": "index.js",
   "dependencies": {
+    "@types/lodash": "^4.14.167",
     "@wmde/wikit-tokens": "^2.0.0-alpha.2",
     "@wmde/wikit-vue-components": "^2.0.0-alpha.2",
     "core-js": "^3.6.5",
+    "lodash": "^4.17.20",
```

Comment:
also, we can use lodash.debounce here. That is what we're also doing in Wikit and that's then maybe a bit less of headache for the sec review :)

Reply:
Done